apps/examples/webserver/webserver_main.c

```diff
@@ -377,60 +379,73 @@ void deregister_callbacks(struct http_server_t *server)
 pthread_addr_t httptest_cb(void *arg)
 {
 	int http_port = 80;
+
 #ifdef CONFIG_NET_SECURITY_TLS
 	int https_port = 443;
 	struct ssl_config_t ssl_config;
+	int auth_mode = MBEDTLS_SSL_VERIFY_REQUIRED;
 #endif
-	struct webserver_input *input;
+	int i = 0;
+	struct webserver_input *input = arg;
 
-	input = arg;
-	if (input->argc != 2) {
+	if (!input && (input->argc != 2 || input->argc != 3)) {
```

Comment:
```
if (!input || (input->argc != 2 && input->argc != 3)) {
```
Isn't it?

Reply:
There are some logic errors, I changed those